scripts/Muon/GUI/ElementalAnalysis2/load_widget/load_run_widget_presenter.py

```diff
@@ -0,0 +1,266 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import copy
+from Muon.GUI.Common import thread_model
+import Muon.GUI.Common.utilities.load_utils as load_utils
+import Muon.GUI.Common.utilities.run_string_utils as run_utils
+from Muon.GUI.Common.utilities.run_string_utils import flatten_run_list
+import Muon.GUI.ElementalAnalysis2.load_widget.load_utils_ea as load_utils_ea
+from mantidqt.utils.observer_pattern import GenericObservable, Observable
+
+
+class LoadRunWidgetPresenterEA(object):
+
+    def __init__(self, view, model):
+        self._view = view
+        self._model = model
+        self._load_thread = None
+
+        self._load_multiple_runs = True
+        self._multiple_run_mode = "Simultaneous"
+
+        self._instrument = self._model.instrument
+        self._view.set_current_instrument(self._instrument)
+
+        self.run_list = []
+
+        self._set_connections()
+        self.enable_notifier = self.EnableEditingNotifier(self)
+        self.disable_notifier = self.DisableEditingNotifier(self)
+
+        self.updated_directory = GenericObservable()
+
+    def _set_connections(self):
+        self._view.on_load_current_run_clicked(self.handle_load_current_run)
+        self._view.on_increment_run_clicked(self.handle_increment_run)
+        self._view.on_decrement_run_clicked(self.handle_decrement_run)
+        self._view.on_run_edit_changed(self.handle_run_changed_by_user)
+
+    def show(self):
+        self._view.show()
+
+    # used with threading
+    def create_load_thread(self):
+        return thread_model.ThreadModel(self._model)
+
+    # used with threading
+    def cancel(self):
+        if self._load_thread is not None:
+            self._load_thread.cancel()
+
+    def set_current_instrument(self, instrument):
+        self._instrument = instrument
+        self._view.set_current_instrument(instrument)
+
+    def disable_loading(self):
+        self._view.disable_load_buttons()
+        self.thread_success = True
+
+    def enable_loading(self):
+        self._view.enable_load_buttons()
+
+    def clear_loaded_data(self):
+        self._view.clear()
+        self._model.clear_loaded_data()
+
+    def clear_data(self):
+        self._model.clear_data()
+
+    @property
+    def workspaces(self):
+        return self._model.loaded_workspaces
+
+    @property
+    def runs(self):
+        return self._model.loaded_runs
+
+    # used by parent widget
+    def update_view_from_model(self, run_list):
+        self.set_run_edit_from_list(run_list)
+
+    def update_multiple_loading_behaviour(self, text):
+        self._multiple_run_mode = text
```

Comment:
Is `text` actually text?

Reply:
This was taken from the Muon Analysis GUI. It refers to the variables CO-ADD or SIMULTANEOUS in the load_widget_presenter.py.

Reply:
can we make the name more helpful?

Reply:
Don't think so - they are the variables for the behaviour when loading multiple runs - either doing it as a co-add or simultaneous load. So updating the multiple loading behaviour seems correct!

Reply:
could we not replace text with multiple_run_mode?